internal/controller/reports.go

```diff
@@ -0,0 +1,67 @@
+package controller
+
+import (
+	"Tradeasy/internal/model"
+	"Tradeasy/internal/services/reports"
+	"fmt"
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+func DailyPendingOrders(c *gin.Context){
+	var rep []model.PendingOrders
```

Comment:
lets use requests structs same as other models.

Reply:
This is a get method and I am directly getting all the orders from pending orders table right ..So, why to use the response struct again and make the same struct as pending orders?